docs/api/text/freqdist.rst

```diff
@@ -5,74 +5,119 @@ Token Frequency Distribution
 
 A method for visualizing the frequency of tokens within and across corpora is frequency distribution. A frequency distribution tells us the frequency of each vocabulary item in the text. In general, it could count any kind of observable event. It is a distribution because it tells us how the total number of word tokens in the text are distributed across the vocabulary items.
 
-.. code:: python
+Note that the ``FreqDistVisualizer`` does not perform any normalization or vectorization, and it expects text that has already be count vectorized.
```

Comment:
This is only a suggestion but consider using a restructuredtext NOTE format here `.. note:: `